app/uk/gov/hmrc/helptosave/models/account/Account.scala

```diff
@@ -108,9 +108,15 @@ object Account extends Logging {
 
   private type ValidOrErrorString[A] = ValidatedNel[String, A]
 
-  private def nsiAccountToBlocking(nsiAccount: NsiAccount): Blocking = Blocking(
-    unspecified = nsiAccount.accountBlockingCode != "00" || nsiAccount.clientBlockingCode != "00"
-  )
+  private def nsiAccountToBlocking(nsiAccount: NsiAccount): Blocking = {
+      def isBlockedFromPredicate(predicate: String ⇒ Boolean): Boolean =
+        predicate(nsiAccount.accountBlockingCode) || predicate(nsiAccount.clientBlockingCode)
+
+    Blocking(
+      unspecified = isBlockedFromPredicate(_ =!= "00"),
+      payments    = isBlockedFromPredicate(s ⇒ s =!= "00" && s =!= "11")
```

Comment:
This will return Blocking.payments = true for unknown blocking codes. That seems reasonable (failsafe) but perhaps it would be worth logging unknown blocking codes when they are encountered.

Reply:
added logging now for unknown blocking codes - I've implemented behaviour where an error is returned if there are unknown blocking codes in alignment with the behaviour with other fields when unknown values are passed onto us